File: parent-group4/pingyougou-web-manager/src/main/webapp/js/service/brandCheckService.js
// 定义服务层:
app.service('brandCheckService',function($http){
    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../brandCheck/findAll.do');
    }
    this.findPage=function(page,rows){
        return $http.get('../brandCheck/findPage.do?page='+page+'&rows='+rows);
    }
    this.search=function(page,rows){
        return $http.get('../brandCheck/search.do?page='+page+"&rows="+rows);
    }
    //审核
    this.updateStatus = function(ids,status){
        return $http.get('../brandCheck/updateStatus.do?ids='+ids+"&status="+status);
    }

});

File: parent-group4/pingyougou-interface/src/main/java/com/itheima/core/BrandCheckService.java
package com.itheima.core;

import com.itheima.core.pojo.good.BrandCheck;
import entity.PageResult;

public interface BrandCheckService {

    void add(BrandCheck brandCheck, String name);

    PageResult<BrandCheck> search(Integer pageNum, Integer pageSize, BrandCheck brandCheck, String name);

    BrandCheck findOne(Long id);

    PageResult search(Integer page, Integer rows);

    void updateStatus(Long[] ids, String status);
}
